bundles/org.openhab.binding.carnet/src/main/java/org/openhab/binding/carnet/internal/handler/CarNetVehicleHandler.java

```diff
@@ -0,0 +1,643 @@
+import org.openhab.carnet.internal.services.CarNetVehicleServiceCharger;
+import org.openhab.carnet.internal.services.CarNetVehicleServiceClimater;
+import org.openhab.carnet.internal.services.CarNetVehicleServiceDestinations;
+import org.openhab.carnet.internal.services.CarNetVehicleServiceRLU;
+import org.openhab.carnet.internal.services.CarNetVehicleServiceStatus;
+import org.openhab.carnet.internal.services.CarNetVehicleServiceTripData;
+import org.openhab.core.library.types.DecimalType;
+import org.openhab.core.library.types.OnOffType;
+import org.openhab.core.thing.Bridge;
+import org.openhab.core.thing.Channel;
+import org.openhab.core.thing.ChannelUID;
+import org.openhab.core.thing.Thing;
+import org.openhab.core.thing.ThingStatus;
+import org.openhab.core.thing.ThingStatusDetail;
+import org.openhab.core.thing.binding.BaseThingHandler;
+import org.openhab.core.thing.binding.builder.ChannelBuilder;
+import org.openhab.core.thing.binding.builder.ThingBuilder;
+import org.openhab.core.thing.type.ChannelKind;
+import org.openhab.core.thing.type.ChannelTypeUID;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.RefreshType;
+import org.openhab.core.types.State;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * The {@link CarNetVehicleHandler} is responsible for handling commands, which are
+ * sent to one of the channels.
+ *
+ * @author Markus Michels - Initial contribution
+ * @author Lorenzo Bernardi - Additional contribution
+ *
+ */
+@NonNullByDefault
+public class CarNetVehicleHandler extends BaseThingHandler implements CarNetDeviceListener {
+    private final Logger logger = LoggerFactory.getLogger(CarNetVehicleHandler.class);
+    private final CarNetTextResources resources;
+    private final CarNetIChanneldMapper idMapper;
+    private final Map<String, Object> channelData = new HashMap<>();
+    private final CarNetTokenManager tokenManager;
+
+    public String thingId = "";
+    private CarNetApi api = new CarNetApi();
+    private @Nullable CarNetAccountHandler accountHandler;
+    private @Nullable ScheduledFuture<?> pollingJob;
+    private int updateCounter = 0;
+    private int skipCount = 1;
+    private boolean forceUpdate;
+    private boolean channelsCreated = false;
+    private boolean testData = false;
+
+    private Map<String, CarNetVehicleBaseService> services = new LinkedHashMap<>();
+    private CarNetServiceAvailability serviceAvailability = new CarNetServiceAvailability();
+    private CarNetCombinedConfig config = new CarNetCombinedConfig();
+
+    public CarNetVehicleHandler(Thing thing, CarNetTextResources resources, CarNetIChanneldMapper idMapper,
+            CarNetTokenManager tokenManager) {
+        super(thing);
+
+        this.resources = resources;
+        this.idMapper = idMapper;
+        this.tokenManager = tokenManager;
+    }
+
+    @Override
+    public void initialize() {
+        logger.debug("Initializing!");
```

Comment:
Can this message be replaced by using the debugger or by increasing the framework's log level? See point 4 https://www.openhab.org/docs/developer/guidelines.html#f-logging